app/src/main/java/com/learnitgirl/myvoc/activities/DictionaryActivity.java

```diff
@@ -0,0 +1,69 @@
+package com.learnitgirl.myvoc.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+import android.widget.Toast;
+
+import com.learnitgirl.myvoc.R;
+import com.learnitgirl.myvoc.database.DictionaryDBHelper;
+
+public class DictionaryActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dictionary);
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        ListView listView = (ListView) findViewById(R.id.dictionary_listview);
+        DictionaryDBHelper dbHelper = new DictionaryDBHelper(this);
+
+        listView.setAdapter(dbHelper.getWordsAdapter(this));
+
+        listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                Toast.makeText(DictionaryActivity.this, position + 1 + "clicked", Toast.LENGTH_SHORT).show();
+            }
+        });
```

Comment:
No, I want to create a floating context menu in order to edit the selected word in ListView. I'm working on it :)